modules/light-clients/08-wasm/types/vm.go

```diff
@@ -72,7 +72,7 @@ func wasmInit(ctx sdk.Context, clientStore storetypes.KVStore, cs *ClientState,
 	}
 	_, err = initContract(ctx, clientStore, cs.CodeHash, encodedData)
 	if err != nil {
-		return errorsmod.Wrapf(err, "call to wasm contract failed")
+		return errorsmod.Wrapf(ErrWasmContractCallFailed, "call to wasm contract failed: %v", err)
```

Comment:
this would be formatted as:
`call to wasm contract failed: <err>: wasm contract call failed`

I generally prefer wrapping the base error so as not to switch up the trace, but I also buy the argument that you want the base error to be from our codebase, but then maybe the wrapped string is unnecessary

Reply:
dropped the wrapped string here since it indeed did not offer much.